documents/converters.go

```diff
@@ -364,3 +360,17 @@ func attrValFromProtocolAttribute(attrType AttributeType, attribute *coredocumen
 
 	return attrVal, err
 }
+
+func timestampToBytes(tm *timestamp.Timestamp) ([]byte, error) {
```

Comment:
maybe this could go in the byteutils folder

Reply:
Agree!